pkg/asset/tls/tls.go

```diff
@@ -24,6 +24,9 @@ const (
 	// ValidityOneDay sets the validity of a cert to 24 hours.
 	ValidityOneDay = time.Hour * 24
 
+	// ValidityOneMinute sets the validity of a cert to 1 month.
+	ValidityOneMonth = ValidityOneDay * 30
```

Comment:
:-/

This was a no-go after discussions. Initial certificate creation was only allowed to be max 24hrs, after which when rotated is extended to the expected 30 days.